src/lib/options/local-storage-area.js

```diff
@@ -47,22 +47,20 @@ export default class LocalStorageArea extends StorageAdapter {
   remove (key) {
     return new Promise((resolve, reject) => {
       try {
-        let result = null
+        const result = null
```

Comment:
I didn't quitely understand - if t won't be changed in the following code (it is const), then why do we need it?
And if finally we resolbe it - then we resolve always null?

Reply:
That's probably an artifact of something. We don't really need a `result` const, so I will remove it.

The idea with promise resoultion is that it is resolved with `true` if we want to remove the item, the item is in the storage, and was removed successfully. If the item we want to remove is not in the storage, it is resolved with `null`. And if there was an error during the removal, then the promise is rejected with the error.